optimum/intel/openvino/configuration.py

```diff
@@ -33,18 +33,17 @@
 logger = logging.getLogger(__name__)
 
 _DEFAULT_4BIT_CONFIGS = {
-    "databricks/dolly-v2-3b": {"bits": 4, "sym": False, "group_size": 128, "ratio": 0.8},
+    "databricks/dolly-v2-3b": {"bits": 4, "sym": False, "group_size": 128, "scale_estimation": True},
     "EleutherAI/gpt-j-6b": {"bits": 4, "sym": False, "group_size": 64},
     "facebook/opt-6.7b": {"bits": 4, "sym": False, "group_size": 64, "ratio": 0.8},
-    "bigscience/bloomz-7b1": {"bits": 4, "sym": False, "group_size": 32, "ratio": 0.6},
     "togethercomputer/RedPajama-INCITE-7B-Instruct": {"bits": 4, "sym": False, "group_size": 128},
     "HuggingFaceH4/zephyr-7b-beta": {
         "bits": 4,
         "sym": True,
         "group_size": 128,
         "ratio": 0.8,
         "dataset": "wikitext2",
-        "awq": True,
+        "quant_method": "awq",
```

Comment:
```suggestion
        "quant_method": OVQuantizationMethod.AWQ,
```

Here and below

Reply:
This requires to replace the `OVQuantizationMethod` definition before `_DEFAULT_4BIT_CONFIGS`. Do you agree with that change?

Reply:
> This requires to replace the `OVQuantizationMethod` definition before `_DEFAULT_4BIT_CONFIGS`. Do you agree with that change?

Yes, sure